company-profiles/fly-io.md

```diff
@@ -0,0 +1,29 @@
+# Fly.io
+
+## Company blurb
+
+[Fly.io](https://fly.io/) is a platform for running full stack apps and databases close to your users (edge computing) all around the world.
+
+## Company size
+
+33 (as of October 2022)
+
+## Remote status
+
+Everyone is remote, but there is also an office in Chicago to who wants to use.
```

Comment:
A readability improvement: 

`Everyone is remote, but there is an office in Chicago for those who would like to use it.`

Reply:
Thanks! I don't speak English natively so thank you for your suggestion, it turned out much better as you proposed.